aws/resource_aws_autoscaling_group_test.go

```diff
@@ -2775,7 +2781,15 @@ resource "aws_autoscaling_group" "bar" {
   	     "GroupPendingInstances",
   	     "GroupTerminatingInstances",
   	     "GroupDesiredCapacity",
-  	     "GroupMaxSize"
+  	     "GroupInServiceInstances",
+  	     "GroupMinSize",
+  	     "GroupMaxSize",
+  	     "GroupPendingCapacity",
+  	     "GroupInServiceCapacity",
+  	     "GroupStandbyCapacity",
+  	     "GroupTotalCapacity",
+  	     "GroupTerminatingCapacity",
+  	     "GroupStandbyInstances"
```

Comment:
The `TestAccAWSAutoScalingGroup_withMetrics` test is currently attempting to verify an API update of this argument and having the two lists be the same will not trigger the API update call. I recommend reverting this section of changes back.

Reply:
Thanks - this has now been reverted